qiita_db/analysis.py

```diff
@@ -200,39 +198,39 @@ def delete(cls, _id):
         QiitaDBUnknownIDError
             If the analysis id doesn't exist
         """
-        with TRN:
+        with qdb.sql_connection.TRN:
```

Comment:
No, it is not worth it to re-do the changes, unless someone thinks
otherwise, in any case lets not consider this blocking.

On (Nov-16-15|19:17), Jose Navas wrote:
> > @@ -200,39 +198,39 @@ def delete(cls, _id):
> >          QiitaDBUnknownIDError
> >              If the analysis id doesn't exist
> >          """
> > -        with TRN:
> > +        with qdb.sql_connection.TRN:
> 
> You're probably correct, but I did a search/replace. Do you think it is needed to change or it is ok as it is?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1529/files#r45018192